Changelog.md

```diff
@@ -23,6 +23,11 @@ rules on making a good Changelog.
 - `delocate-wheel` is now more strict with platform tags and will no longer allow
   a wheel to be incompatible with its own tags. #198
 
+### Fixed
+
+- Fixes deleting duplicated rpaths when sanitize-rpaths option is turned on.
+  [#208](https://github.com/matthew-brett/delocate/pull/208)
```

Comment:
Less vague changelog item.
```suggestion
- `--sanitize-rpaths` was not removing duplicate rpaths.
  [#208](https://github.com/matthew-brett/delocate/pull/208)
```

Reply:
Changed this to '`--sanitize-rpaths` was failing with duplicate rpaths.' as it was failing with an exception from subprocess.